frontend/src/components/MomentPreview.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import '../styles/MomentPreview.css';
+
+const MomentPreview = props => {
```

Comment:
This component is exactly what a Moment is, why not keep it Moment for reusability?

Reply:
The way I see it is that the non-preview or full moment component would include the associated captions. What do you think?

Reply:
Meaning the component shown on the Moment page would not be the "moment-preview" component with added captions, but an entirely new component?

Reply:
My understanding is that when the moment will be viewed in its own page with its associated captions, it will have a different layout, possibly including more information related to it. However, it would probably make more sense to have a single moment component having multiple layout options a parameter.